docs/spec/spec.md

```diff
@@ -17,96 +17,97 @@ If you have any feedback or suggestions about the library, start a discussion vi
 The conforming implementation of the specification is released and included in the distribution. Any deviation from the specification is considered a bug.
 
 ## Contents
+
 1. [Overview](#1-overview)
 2. [Hash](#2-hash)
-   * 2.1. [MD5](#21-md5)
-   * 2.2. [SHA1](#22-sha1)
-   * 2.3. [SHA256](#23-sha256)
-   * 2.4. [SHA384](#24-sha384)
-   * 2.5. [SHA512](#25-sha512)
-   * 2.6. [CRC32B](#26-crc32b)
-   * 2.7. [KECCAK256](#27-keccak256)
+   - 2.1. [MD5](#21-md5)
```

Comment:
@randilt shall we update the code with previous formatting?
replace them with '*' instead of '-'.
We follow this same convention across all library modules.